src/markdown/features/routes.md

```diff
@@ -0,0 +1,77 @@
+# Defining Routes
+
+Routes are defined in the `lenra.yml` file under the `lenraRoutes` property. The `lenraRoutes` property is an array of objects, where each object represents a route. Each route object has two properties:
+
+- `path` - Defines the URL path that corresponds to the route.
+- `widget` - Defines the view that is associated with the route.
+
+## lenra.yml example
+
+```js
```

Comment:
Avoid language specific examples when you can.

Just describe a JSON manifest example with routes.